homeassistant/components/zha/core/channels/manufacturerspecific.py

```diff
@@ -13,6 +13,9 @@
     UNKNOWN,
 )
 from .base import ZigbeeChannel
+import logging
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
I think we can remove this. I put that in there to check my work. I don't think it is used in the file.

Reply:
Ok perfect, went ahead and removed that piece